main.go

```diff
@@ -221,12 +224,13 @@ func main() {
 	}
 
 	if err = (&controller.ClusterMonitoringReconciler{
-		Client:                 mgr.GetClient(),
-		ManagementCluster:      managementCluster,
-		HeartbeatRepository:    heartbeatRepository,
-		PrometheusAgentService: prometheusAgentService,
-		MimirService:           mimirService,
-		MonitoringConfig:       monitoringConfig,
+		Client:                     mgr.GetClient(),
+		ManagementCluster:          managementCluster,
+		HeartbeatRepository:        heartbeatRepository,
+		PrometheusAgentService:     prometheusAgentService,
+		MimirService:               mimirService,
+		MonitoringConfig:           monitoringConfig,
+		BundleConfigurationService: bundle.NewBundleConfigurationService(mgr.GetClient(), monitoringConfig),
```

Comment:
Why do we need a pointer here ?

Reply:
We could do without but that's not really changing anything

Reply:
Consistency in the code like the other services

Reply:
it's fixed :)